docs/visualization/wcsaxes/overlays.rst

```diff
@@ -326,9 +326,20 @@ In the case where you are making a plot of a celestial image, and want to plot a
    :align: center
 
     from astropy import units as u
+    from astropy.coordinates import SkyCoord
     from astropy.visualization.wcsaxes import SphericalCircle
 
+
     r = SphericalCircle((266.4 * u.deg, -29.1 * u.deg), 0.15 * u.degree,
                          edgecolor='yellow', facecolor='none',
                          transform=ax.get_transform('fk5'))
+
     ax.add_patch(r)
+
+    #The following lines show the usage of a SkyCoord object as the input.
+    skycoord_object = SkyCoord(266.4 * u.deg, -28.7 * u.deg)
```

Comment:
Last comment from my side, I promise. For the test, we put the circles on top of each other of that we can do the assert statement. For the docs, won't it be better to have different circles so that both can be seen? Or at least a comment in the code something like: This command makes the same circle so you won't see it in the plot because they are on top of each other"?

Reply:
The circles in the docs can be seen seperately. Are you suggesting to have them on top of each other?

Reply:
Ooh, OK, I see. Sorry.